.github/workflows/publish-package.yml

```diff
@@ -40,5 +34,36 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+      - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.WDIO_TOKEN }}
+
+      - name: Install packages
+        run: yarn install
+
       - name: Release new package
-        run: standard-version --dry-run
+        run: |
+          git config user.name ${{ secrets.USERNAME }}
+          git config user.email "${{ secrets.MAIL }}"
+          yarn release
+
+      - name: push changes
+        run: git push --follow-tags origin HEAD
+
+      - uses: JS-DevTools/npm-publish@v1
+        with:
+          token: ${{ secrets.NPM_TOKEN }}
+
+  slack_notification:
+    needs: [ release_package ]
```

Comment:
this will be executed only in case of success. need to send a message also in case of failure

Reply:
done